server/handlers.go

```diff
@@ -98,7 +132,7 @@ func getLogsStartTime(api plugin.API) (time.Duration, error) {
 	}
 
 	if seconds <= 0 {
-		return defaultLogsStartTime, nil
+		return time.Duration(defaultLogsStartTime) * time.Second, nil
 	}
```

Comment:
I think it won't be triggered now (we have default values set on activation and similar check above).

The same applies to other `get...` functions.

Reply:
Looks like I forgot to push this change, done.